css/main.css

```diff
@@ -65,3 +65,187 @@ header a {
   color: #172b4d;
   margin-bottom: 58px;
 }
+
+.container-2 {
+  margin-left: 26px;
+  margin-right: 22px;
+}
+
+.cricle {
+  width: 80px;
+  height: 80px;
+  margin-top: 40px;
+  border-radius: 100%;
+  background: white;
+  box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  row-gap: 24px;
+}
+
+.work-section h2 {
```

Comment:
- [ ] It would be nice to include the horizontal green bar in your project just like in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design. Cheers 😃  

## Your Design 
![image](https://user-images.githubusercontent.com/60618877/176966686-4af2d1b9-4af9-44ca-8394-dc78e181e5aa.png)

## Figma Design 
![image](https://user-images.githubusercontent.com/60618877/176966359-f98f6790-e3d8-42d5-9adb-8dd976267b77.png)

Reply:
@sumairq thanks man this one i did not notice it at all thanks for notifying me already resolved